test/runner.py

```diff
@@ -392,6 +392,8 @@ def parse_args():
                       help='Automatically update test expectations for tests that support it.')
   parser.add_argument('--browser',
                       help='Command to launch web browser in which to run browser tests.')
+  parser.add_argument('--headed', action='store_true',
+                      help='Run browser tests in regular (non-headless) mode.')
```

Comment:
Does this have a default of None, i.e. not headed, i.e. headless? (the reverse of the current behavior)

Reply:
Should we keep the default as headed for now to avoid breakage and have CI use `--headless`?

Reply:
How about `--headless=0` AKA `--no-headless`.

Then we can use `default=true` if we want it to be the default...

Reply:
It seems people are leaning towards "headed" by default, so I changed it back. The env var `EMTEST_HEADLESS` or `--headless` make it easy enough to turn it on.